src/components/Lend/LoanSearch/ChallengeCallout.vue

```diff
@@ -109,6 +106,7 @@ export default {
 	data() {
 		return {
 			mdiArrowTopRight,
+			participantsMessage: 'You are on your way to supporting the team challenge!'
```

Comment:
This message is only when there are no other participants. So this message should be in the else of the tertiary that used to be in `participantsMessage`

Reply:
I think your latest changes are close, but I'm guessing we only want to show the borrower name if there are more than one participant, since the new message probably doesn't make sense with the borrower name just at the end.